src/components/ExpensiTextInput/BaseExpensiTextInput.js

```diff
@@ -167,7 +175,7 @@ class BaseExpensiTextInput extends Component {
                             {hasLabel ? (
                                 <>
                                     {/* Adding this background to the label only for multiline text input,
-                                    to prevent text overlaping with label when scrolling */}
+                                    to prevent text overlapping with label when scrolling */}
                                     {this.props.multiline && <View style={styles.expensiTextInputLabelBackground} />}
```

Comment:
Could you please add `pointerEvents="none"` to this `View`. it fixes one bug, not from your PR.

Reply:
```
    expensiTextInputLabelBackground: {
        position: 'absolute',
        top: 0,
        width: '100%',
        height: 25,
        backgroundColor: themeColors.componentBG,
        borderTopRightRadius: variables.componentBorderRadiusNormal,
        borderTopLeftRadius: variables.componentBorderRadiusNormal,
        pointerEvents: 'none',
    },
```

Reply:
I think pointerEvents=none is used as a prop. Styles only for the web.